src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -68,18 +68,29 @@ default String getAsTextHidePrivate() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName());
         if (!getPhone().isPrivate()) {
-            builder.append(" Phone: ").append(getPhone());
+            getPrintableString(getPhone());
         }
         if (!getEmail().isPrivate()) {
-            builder.append(" Email: ").append(getEmail());
+            getPrintableString(getEmail());
         }
         if (!getAddress().isPrivate()) {
-            builder.append(" Address: ").append(getAddress());
+            getPrintableString(getAddress());
```

Comment:
While this works, seems to be an abusing of the method. :-p
It's easier to understand if you used `getAddress().getPrintableString() + " "` instead?